app/services/analytics_events.rb

```diff
@@ -1053,9 +1053,8 @@ def idv_doc_auth_submitted_image_upload_vendor(
     exception:,
     state:,
     state_id_type:,
-    async:,
-    attempts:,
-    remaining_attempts:,
+    async:, submit_attempts:,
+    remaining_submit_attempts:,
```

Comment:
I would undo this spacing, I think separate lines are clearer
```suggestion
    async:,
    submit_attempts:,
    remaining_submit_attempts:,
```